File: AmazonDemo.java
import org.openqa.selenium.By;
import org.openqa.selenium.WebDriver;
import org.openqa.selenium.WebElement;
import org.openqa.selenium.chrome.ChromeDriver;

public class AmazonDemo {

	public static void main(String[] args) {
		System.setProperty(
				"webdriver.chrome.driver",
				"./drivers/chromedriver"
		);
		WebDriver driver = new ChromeDriver();
		driver.get("http://www.amazon.com/");
		
		// I am in Amazon homepage.
		// I need to find the search input box to type the keyword.
		WebElement input = driver.findElement(By.id("twotabsearchtextbox"));
		input.sendKeys("selenium webdriver");
		WebElement searchIcon =driver.findElement(By.cssSelector("#nav-search > form > div.nav-right > div > input"));
		
		searchIcon.click();
		
		// I am in the result page now!
		WebElement status = driver.findElement(By.id("s-result-count"));
		System.out.println(status.getText());
		driver.quit();
		}
	}

File: amazon_demo.py
import os

from selenium import webdriver

chromedriver = './driver/chromedriver'
os.environ['webdriver.chrome.driver'] = chromedriver
driver = webdriver.Chrome(chromedriver)
driver.get('http://www.amazon.com/')

input_elem = driver.find_element_by_id('twotabsearchtextbox')
input_elem.send_keys("selenium webdriver")
search_icon = driver.find_element_by_css_selector('#nav-search > form > div.nav-right > div > input')
search_icon.click()

status = driver.find_element_by_id('s-result-count')
print(status.text)
driver.quit()
